src/openrct2/interface/Viewport.cpp

```diff
@@ -836,9 +836,26 @@ void viewport_render(
 #endif
 }
 
-static void viewport_fill_column(paint_session* session)
+static void viewport_fill_column(paint_session* session, std::vector<paint_session>* sessions, size_t index)
```

Comment:
The thing between session and sessions is unclear the same applies to the index, would be better to give it more concrete names.